server/api/repos/[repo_id]/indexing.post.ts

```diff
@@ -0,0 +1,26 @@
+export default defineEventHandler(async (event) => {
```

Comment:
@aaa006bd I think the cloning and new indexing endpoint can still be one. I just wanted to separate the:
- add the repo to the db
- clone / update the repo and index it

tasks to be separated, so that we can call the 2. again if we want to re-index.

Reply:
@aaa006bd Ahh sorry, yeah the description was quite bad 🙈